_posts/2021-01-04-2020-reading-in-review.md

```diff
@@ -0,0 +1,250 @@
+---
+layout: post
+title: "2020 Reading in Review"
+date: "2021-01-04 23:30:11 -0600"
+---
+
+Welp. _That_ was certainly… a year. Let's talk about books! I read _way_ fewer
+things this year than I did [last year](/2020/01/06/2019-reading-in-review), for
+reasons that I'm sure many of us did way less of shit in 2020 than they
+expected.
+
+As a reminder: I keep a one-line entry in a journal and record the title, author
+and date that I finish each thing. I don't include comics or childrens' books.
+
+
+## The Numbers
+
+Let's do some numbers, even thought they don't mean a ton. The parenthetical
+numbers are 2019's stats.
+
+**Number of Things Read:** 16 (47)
+
+**Reading Time** (in days including weekends, during which I generally don't
+read):
+* Maximum: 174\* days (_The Phoenix Princess_)
+* [Median](https://en.wikipedia.org/wiki/Median): 11 days (_Jacobin 35 and
+  _Catalyst Vol. 3 No. 3_)
+* Minimum: 1† day (_The Five Dysfunctions of a Team)
+* [Mode](https://en.wikipedia.org/wiki/Mode_(statistics)): 4 days (_The
+  Consuming Fire_, _Books and Bone_ and _Obsidio_)
+
+\* This maximum is in error. Because I don't record the starting date for books,
+if there's a gap, it gets added to the next book I read. Normally that's not a
+big problem because gaps are a few days at most. In April 2020, however, I was
+part way through _Who Fears Death_ by Nnedi Okorofor when I realized I'd been
+"reading" it for weeks and I probably wasn't in the right emotional place for
+such an intense book. So I picked up _How to Talk so Kids Will Listen & Listen
+so Kids Will Talk_ by Adele Faber and Elaine Mazlish because my mom gifted it to
+me. That proved too dry for the emotional and mental state I was in. So
+eventually, I picked up _The Phoenix Princess_ and absolutely blazed through it.
+I just have no idea how fast I read it, so I can't correct the numbers. Also,
+FWIW, I _do_ intend to return to those other books. Just… later.
+
+† I think this minimum is also probably in error. I do remember that the first,
+maybe two thirds of the book were incredibly fast to read, but I think the last
+third took me at least another day? I think it's likely that I was reading this
+concurrently with _Jacobin No. 35_ for some reason.
+
+
+## Things I Read in 2020
+
+### _Spinning Silver_ by Naomi Novik (2020-01-19)
+
+After reading this, I have beef with Novik as a writer. I like her writing style
+a lot and her world building and main characters are great. And in _Uprooted_, I
+excused some of this stuff as, like, a one-off problem. But this book shows that
+it's a pattern and so I'm almost certainly done with her as an author. The deal
```

Comment:
The ordering here is confusing.  I had to reread the first 2 sentences to realize I didn't miss anything.  "this stuff" felt like it was referencing something I should have already known.